client/state/happychat/README.md

```diff
@@ -1,13 +1,23 @@
 Happychat State
 ==========
 
-A module for managing user state.
+Happychat state shape is:
+
+- connection
+  - error: one of the HAPPYCHAT_CONNECTION_ERROR_* [constants](./constants.js)
+  - isAvailable: whether the Happychat service is accepting new chats.
+  - status: one of the HAPPYCHAT_CONNECTION_STATUS_* [constants](./constants.js)
+- chatStatus
+- lastActivityTimestamp
+- lostFocusAt
+- message: current message as typed by the customer in the happychat client.
+- timeline: array of timeline messages, as received from the Happychat service.
+- geoLocation
```

Comment:
You should probably also change this to `user -> geoLocation` after #18739 is merged

Reply:
My plan is to update docs all at once as a final step.